### DISCUSSION TITLE
Can the RaspAP host provide NTP support?

### DISCUSSION THREAD

**Discussion by WillMarchant** [RaspAP/raspap-webgui#1553] [Q&A]
I'm hoping someone will be kind enough to provide a tutorial on configuring the machine running RaspAP to provide clients connecting to the access point NTP support when there is no Internet connection.  Thanks in advance.

**Comment by billz** [comment]
This is currently under development as an optional add-on for RaspAP. I don't have a comprehensive tutorial on hand, however you can install it with `sudo apt install ntp`. Thereafter, consult the man pages with `man ntpd` and `man ntpq`.

**Comment by billz** [comment]
The NTP server feature is nearly code complete. @WillMarchant glad to let you evaluate it for your needs. 

<img width="500" alt="ntp-feature" src="https://github.com/RaspAP/raspap-webgui/assets/229399/c1fcb5cc-c87f-4059-8abe-ab2ac646d504">

**Reply by WillMarchant** [reply]
I'd love to test that out, Bill!  What should I do next?

**Reply by billz** [reply]
Look for an invite from GitHub to the private Insiders repository. After you accept it, you can then [upgrade](https://docs.raspap.com/insiders/#upgrading) your existing install or create a new one. Take note of [authenticating](https://docs.raspap.com/insiders/#authentication) with GitHub. You're welcome to share your observations here or drop a message to bill [at] raspap.com.

NTP doc page is here https://docs.raspap.com/time/
